pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -212,7 +224,7 @@ in (buildEnv {
 
     perl `type -P mktexlsr.pl` ./share/texmf
     ${bin.texlinks} "$out/bin" && wrapBin
-    (perl `type -P fmtutil.pl` --sys --all || true) | grep '^fmtutil' # too verbose
+    perl `type -P fmtutil.pl` --sys --all | grep '^fmtutil' # too verbose
```

Comment:
Little known fact: `set -e` doesn't catch things like `false | cat`

Reply:
Uhm, the intention is for the build to fail if `fmtutil` fails. At the moment, the build does fail if e.g. `fmtutil.cnf` contains formats for which the binaries are not available.